src/client/components/CardCvViewSearchComponent/CardCvViewSearchComponent.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import unknownUser from '../../assets/images/portraits/unknown-black.jpg';
+import './CardCvViewSearchComponent.styles.css';
+
+export const CardCvViewSearchComponent = ({
+  profileImageUrl,
+  fullName,
+  position,
+  cvTitle,
+  fileUrl,
+  createdDate,
+}) => {
+  const parsedDate = (date) =>
+    new Date(date).toLocaleDateString('en-GB', { timeZone: 'UTC' });
```

Comment:
You shouldn't have to provide a timezone, `new Date(date).toLocaleDateString();` should be sufficient.

Reply:
Ok, I have updated the files. Thank you for the feedback @pdsorensen